css/webots-doc.css

```diff
@@ -283,9 +283,16 @@
   margin: 0px;
 }
 
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 100;
+  src: local("fonts/Roboto"), local("fonts/Roboto"), url("fonts/Roboto.ttf") format("truetype");
```

Comment:
`local("fonts/Roboto")` seems redundant, is that correct?

Reply:
You're right, it seems this can be removed. I will make a new PR to do this now.